_includes/prod_deployment/synchronize-clocks.md

```diff
@@ -0,0 +1,145 @@
+CockroachDB requires moderate levels of [clock synchronization](recommended-production-settings.html#clock-synchronization) to preserve data consistency. For this reason, when a node detects that its clock is out of synch with at least half of the other nodes in the cluster by 80% of the maximum offset allowed (500ms by default), it spontaneously shuts down. This avoids the risk of consistency anomalies, but it's best to prevent clocks from drifting too far in the first place by running clock synchronization software on each node.
+
+{% if page.title contains "Digital Ocean" or page.title contains "Manual" %}
+
+[NTP](http://www.ntp.org/) should keep offsets in the single-digit milliseconds, so that software is featured here, but other methods of clock synchronization are suitable as well.
+
+1. SSH to the first machine.
+
+2. Disable `timesyncd`, which tends to be active by default on some Linux distributions:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ sudo timedatectl set-ntp no
```

Comment:
I can confirm that this disables `systemd-timesyncd.service` permanently (as opposed to just stopping it). This is mentioned in the [timedatectl docs](https://www.freedesktop.org/software/systemd/man/timedatectl.html):
```
This command is hence mostly equivalent to: systemctl enable --now systemd-timesyncd.service
and systemctl disable --now systemd-timesyncd.service, but is protected by a different access policy.
```

Reply:
Great, thanks.